playground/DetailsPage.tsx

```diff
@@ -475,7 +479,7 @@ export function DetailsPage() {
         {
           content: 'View',
           // eslint-disable-next-line no-console
-          onAction: () => console.log('view'),
+          onAction: () => console.log(previewValue),
```

Comment:
Web has a preview action on product pages where the url hash changes on any update to any field on the page. That triggers a re-render of this page action and therefore all page actions